### ISSUE TITLE
Fictionzone.net

### ISSUE THREAD

**Issue by BlackStar0000** [dipu-bd/lightnovel-crawler#2474]
Please add fictionzone.net

**Comment by dipu-bd** [comment]
Keeping this as the single open request for `fictionzone.net`; the other reports are closed as duplicates of it.

**Status: blocked on something a source cannot work around.** What has been ruled out:

- The API (`api2.fictionzone.net/api/v1`) exposes **no chapter-list route**. 162 candidate paths were probed; the broker answers `Cannot GET <path>` for anything unregistered, so this is an enumeration rather than guesswork. Only `platform/browse`, `platform/novel-details` and `novels` exist, and `novel-details` returns `total_chapters` without a list.
- `sitemap.xml` holds static pages only — no novels, no chapters.
- The Nuxt payload route (`/novel/<slug>/_payload.json`) answers **403**.
- The reader page (`/omniportal/<slug>/<chapter>`) loads in a real browser but **never hydrates**: 167 KB of shell around 701 characters, which is the cookie banner.
- `/ai/search` answers **401**, which points at the reading path requiring a signed-in token.

If you know a chapter URL that returns text without JavaScript, that would unblock it — please post one here.
